lib/cn/cn.c

```diff
@@ -968,6 +969,20 @@ cn_open(
         }
     }
 
+    /* Put the tree's node list into edge key order.
+     */
+    rn = route_map_first_node(cn->cn_tree->ct_route_map);
+    while (rn) {
+        tn = route_node_tnode(rn);
+        if (!tn)
+            abort();
+
+        list_del(&tn->tn_link);
+        list_add_tail(&tn->tn_link, &cn->cn_tree->ct_nodes);
+
+        rn = route_node_next(rn);
+    }
```

Comment:
Why does this list need to be in order?

Reply:
On. NVM.  I see in the PR comment.

Reply:
Life is so much better now that this list is sorted :-)